addons/mail/models/mail_alias.py

```diff
@@ -229,11 +229,24 @@ def open_parent_document(self):
         }
 
     def _get_alias_bounced_body_fallback(self, message_dict):
+        contact_description = self._get_alias_contact_description()
+        default_email = self.env.company.partner_id.email_formatted if self.env.company.partner_id.email else self.env.company.name
         return Markup(
-            _("""<p>Hi,<br/>
-The following email sent to %s cannot be accepted because this is a private email address.
-Only allowed people can contact us at this address.</p>""")
-        ) % self.display_name
+            _("""<p>Dear Sender,<br /><br />
+The message below could not be accepted by the address %(alias_display_name)s.
+Only %(contact_description)s are allowed to contact it.<br /><br />
+Please make sure you are using the correct address or contact us at %(default_email)s instead.<br /><br />
+Kind Regards,</p>""",
+              alias_display_name=self.display_name,
+              contact_description=contact_description,
+              default_email=html_escape(default_email)
```

Comment:
Mind moving all of that outide the `Markup` call? And the `html_escape` is then unnecessary: `markupsafe.Markup` will automatically escape whatever is being formatted in.

Reply:
@awa-odoo asked me the inverse. You are playing with my feelings, especially that I did not find any schokobon last Friday.

Reply:
> @awa-odoo asked me the inverse. 

Sounds like @awa-odoo is outdated.

Passing the arguments as part of the `_()` makes sense when there's only that, but when using a `Markup` it's much better to let *it* perform the formatting, so it can also escape everything which is not already escaped (aka not already a `Markup` object).

> You are playing with my feelings, especially that I did not find any schokobon last Friday.

That's because you never looked for them, they were literally on a desk corner the entire day, and I left them there when I, well, left.

Reply:
:older_man: What year is this?

Reply:
The closing years of the 41st millenium.